std/algorithm/searching.d

```diff
@@ -1792,6 +1796,7 @@ if (isRandomAccessRange!R1 && isBidirectionalRange!R2
     }
     // Stage 2: linear find
     size_t scout = needleLength - 1;
+outer:
```

Comment:
The no-indentation-for-labels style is common in C and my vim auto-indent defaults to this. The label-before-loop style is common in Java, I think.